src/Ext/Techno/Hooks.cpp

```diff
@@ -471,3 +472,59 @@ DEFINE_HOOK(0x4CD4E1, FlyLocomotionClass_Update_LayerUpdate, 0x6)
 
 	return 0;
 }
+
+DEFINE_HOOK(0x7037F7, TechnoClass_Cloak_CloakAnim, 0x5)
+{
+	GET(TechnoClass* const, pThis, ESI);
+
+	if (const auto pAnimType = TechnoTypeExt::ExtMap.Find(pThis->GetTechnoType())->CloakAnim.Get(RulesExt::Global()->CloakAnim))
```

Comment:
```suggestion
    const auto pExt = TechnoTypeExt::ExtMap.Find(pThis->GetTechnoType());
    if (!pExt )
        return 0;
	if (const auto pAnimType = pExt->CloakAnim.Get(RulesExt::Global()->CloakAnim))
```

If we fail to find TechnoTypeExt (for some reason), we'll try to access null and crash.

Reply:
OK